tutorials/Numerics/TimeStepping/explicit_lsrk.jl

```diff
@@ -0,0 +1,190 @@
+# # [Single-rate Explicit Timestepping](@id Single-rate-Explicit-Timestepping)
+
+# In this tutorial, we shall explore the use of explicit Runge-Kutta
+# methods for the solution of nonautonomous (or non time-invariant) equations.
+# For our model problem, we shall reuse the rising thermal bubble
+# tutorial. See its [tutorial page](@ref Rising-Thermal-Bubble-Configuration)
+# for details on the model and parameters. For the purposes of this tutorial,
+# we will only run the experiment for a total of 100 simulation seconds.
+
+using ClimateMachine
+const clima_dir = dirname(dirname(pathof(ClimateMachine)));
+include(joinpath(
+    clima_dir,
+    "tutorials",
+    "Numerics",
+    "TimeStepping",
+    "tutorial_risingbubble_config.jl",
+))
+
+FT = Float64;
+
+# After discretizing the spatial terms in the equation, the semi-discretization
+# of the governing equations have the form:
+
+# ``
+# \begin{aligned}
+#     \frac{\mathrm{d} \boldsymbol{q}}{ \mathrm{d} t} &= M^{-1}\left(M S +
+#     D^{T} M (F^{adv} + F^{visc}) + \sum_{f=1}^{N_f} L^T M_f(\widehat{F}^{adv} + \widehat{F}^{visc})
+#     \right) \equiv \mathcal{T}(\boldsymbol{q}).
+# \end{aligned}
+# ``
```

Comment:
Should we define the notations in the above equation?

Reply:
I agree those should be explained. However perhaps it is not in the scope of this PR. We should have a section filled with the mathematical formulation chosen. Perhaps this [section](https://clima.github.io/ClimateMachine.jl/previews/PR1938/APIs/Numerics/DGMethods/DGMethods/#Mathematical-Formulation) that I am just seeing now it says "to be filled"